apps/communications/contacts/test/marionette/multiple_select_test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+/* global MozActivity */
+
+var Contacts = require('./lib/contacts');
+var Sms = require('./lib/sms');
+var assert = require('assert');
+var fs = require('fs');
+
+marionette('Contacts > MultipleSelect', function() {
+  var client = marionette.client(Contacts.config);
+  var subject;
+  var smsSubject;
+  var selectors;
+
+  setup(function() {
+    subject = new Contacts(client);
+    smsSubject = new Sms(client);
+    selectors = Contacts.Selectors;
+  });
+
+  suite('Import', function() {
+    function getNumberOfContacts() {
+      return client.executeScript(function() {
+        var selector = 'li.contact-item[data-search]';
+        var contactNodes = document.querySelectorAll(selector);
+        return contactNodes.length;
+      });
+    }
+
+    setup(function() {
+      smsSubject.launch();
+    });
+
+    test('Importing all contacts', function() {
+      client.executeScript(function(vCardFile) {
+        new MozActivity({
+          name: 'open',
+          data: {
+            type: 'text/vcard',
+            filename: 'vcard_21_multiple.vcf',
+            blob: new Blob([vCardFile], {type: 'text/vcard'}),
+            allowSave: true
+          }
+        });
+      }, [fs.readFileSync(__dirname + '/data/vcard_21_multiple.vcf', 'utf8')]);
+
+      client.switchToFrame();
```

Comment:
be careful here. if the purpose of the test is to actually check that the contacts are actually imported you would need to call the mozContacts API to check that after importing the getCount() of mozContacts is 2. So getNumberOfContacts should be implemented on a different way